tools/maf_clean_up.py

```diff
@@ -0,0 +1,177 @@
+input_maf = open ("data_mutations_extended.txt", "r")
+header_line = "";
+
+# search for header row
+for line in input_maf:
+	values = line.split("\t");
+	if values[0].upper() == "HUGO_SYMBOL":
+		header_line = line;
+
+headers = header_line.rstrip().split("\t");
+keep_col_index_arr = []; #Index of columns to keep
+input_maf.close();
+
+# iterate through all column headers and record the index of targeted ones
+count = 0;
+exon_number_index = -1;
+exon_index = -1;
+intron_index = -1;
+oncotator_index_arr = [];
+for header_item in headers:
+	if header_item.upper() == "EXON_NUMBER":
+		exon_number_index = count;
+	elif header_item.upper() == "EXON":
+		exon_index = count;
+	elif header_item.upper() == "INTRON":
+		intron_index = count;
+	elif header_item[0:len("ONCOTATOR")] == "ONCOTATOR":
+		oncotator_index_arr.append(count);
+	else:
+		keep_col_index_arr.append(count);
+	count += 1;
+
+print "INDEX OF COLUMNS TO DROP >>>>>>>>>>>>>>";
+print "INTRON: " + str(intron_index);
+print "EXON_NUMBER: " + str(exon_number_index);
+print "EXON: " + str(exon_index);
+print "ONCOTATOR: " + str(oncotator_index_arr)[1:-1] ;
```

Comment:
str(oncotator_index_arr)[1:-1] can be written as ' '.join(oncotator_index_arr)

or if it is an array of int
' '.join(str(index) for index in oncotator_index_array)

Reply:
fixed